src/notebook/notebook/widget.ts

```diff
@@ -716,11 +832,31 @@ class Notebook extends StaticNotebook {
   }
 
   /**
+   * Update the active cell when a cell is added or removed.
+   */
+  private _updateActiveCell(): void {
+    let count = this.childCount();
```

Comment:
I consolidated the logic in c09195b.  The active cell cannot be set to -1, it gets clamped to 0, we always have one active cell if there is more than one cell.